src/components/navBar/navBar.module.scss

```diff
@@ -8,28 +8,71 @@ $thickness: 3px;
   align-items: center;
   overflow: hidden;
   flex-wrap: wrap;
+  justify-content: space-between;
 
-  a {
-    display: block;
-    color: #ffffff;
-    font-weight: bolder;
-    text-align: center;
-    padding: 35px 2*($offset+$thickness)-1;
-    text-decoration: none;
+  div {
+    display: flex;
 
-    &:hover {
+    a {
+      display: block;
+      color: #ffffff;
+      font-weight: bolder;
+      text-align: center;
+      padding: 35px 2 * ($offset + $thickness)-1;
```

Comment:
Could you remove the space after 2, it creates confusion.

Reply:
done